gradle.properties

```
@@ -20,5 +20,5 @@ android.minSdk=21
 android.targetSdk=23
 android.compileSdk=34
 
-version=0.5.0
+version=0.6.0-SNAPSHOT
```

We append `-SNAPSHOT` in the build itself [over here](https://github.com/open-telemetry/opentelemetry-android/blob/main/build.gradle.kts#L29) and don't expect it to be a literal in the version string.

```suggestion
version=0.6.0
```

---

I'll create a follow-up PR to fix the build process for next time.